packages/uniforms-material/src/ErrorField.tsx

```diff
@@ -1,8 +1,9 @@
-import { PropTypes, useTheme } from '@material-ui/core';
+import type { PropTypes } from '@material-ui/core';
```

Comment:
I think we are not using `import type` anywhere. Let's stick to `import`s, at least for now.

Reply:
I added this as you were concerned whether or not the whole MUI package would be emitted: https://github.com/vazco/uniforms/pull/993#discussion_r684961200

This ensures that it will not be included in the compiled code and I think overall is a good practice in cases like these that doesn't require us to manually check whether or not every import will get tree shaken (as I haven't found this being exported from anywhere else).

Reply:
I understand, but then I'd rather use `import type` everywhere. Ideally, ESLint or `tsc` should tell us that an import should be type-only.